files/default/rubocop.yml

```diff
@@ -4,3 +4,6 @@ AllCops:
     - '**/Cheffile'
   Exclude:
     - 'metadata.rb'
+
+Style/NumericLiteralPrefix:
+  Enabled: false
```

Comment:
Would using `EnforcedOctalStyle` => `zero_only` be a better option for this instead? See [upstream docs](http://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/NumericLiteralPrefix).